server/src/main/java/org/apache/gravitino/server/web/rest/PermissionOperations.java

```diff
@@ -216,8 +229,10 @@ public Response grantPrivilegeToRole(
   @Produces("application/vnd.gravitino.v1+json")
   @Timed(name = "revoke-privilege-from-role." + MetricNames.HTTP_PROCESS_DURATION, absolute = true)
   @ResponseMetered(name = "revoke-privilege-from-role", absolute = true)
+  @AuthorizationExpression(expression = "METALAKE::OWNER || METALAKE::MANAGE_GRANTS")
```

Comment:
If the user is the owner of the metadata object, he can revoke the privilege from any role.

Reply:
Does this mean that the Owner of the Schema can also manage permissions for all tables in the same schema?